src/components/SortBy/SortBy.spec.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { SortBy } from '../SortBy/SortBy';
+import constants from '../../constants';
+
+describe('< SortBy />', () => {
```

Comment:
I would also add test cases which click buttons and check that props has been called